capstone/src/main/java/com/google/step/data/RestaurantPageViews.java

```diff
@@ -44,8 +46,22 @@ public boolean equals(Object other) {
             return false;
         }
         RestaurantPageViews restaurantPageViews = (RestaurantPageViews) other;
+        if (this.hashCode() != other.hashCode()) {
```

Comment:
nit: You don't really need this, since now you're just reading all of the fields twice.

Reply:
Done